.github/workflows/Dockerfile.ci.debian

```diff
@@ -46,10 +46,13 @@ RUN cp target/lldap /lldap/ && \
           web/static \
           /lldap/app/
 
+WORKDIR /lldap
 RUN set -x \
     && for file in $(cat /lldap/app/static/libraries.txt); do wget -P app/static "$file"; done \
     && for file in $(cat /lldap/app/static/fonts/fonts.txt); do wget -P app/static/fonts "$file"; done \
     && chmod a+r -R .
+RUN ls -al app/static && \
```

Comment:
Same here, we want to merge the stages.
And we need to use index_local as well.